.circleci/config.yml

```diff
@@ -27,3 +41,32 @@ jobs:
           name: Run Scala/Java and Python tests
           command: |
             pipenv run python run-tests.py
+
+  deploy:
```

Comment:
what is this for? is this for testing the deployment on a regular basis or for actually deploying?

Reply:
This is for actually deploying. What does release automation look like now? I propose to create a process that packages and uploads to pypi as well that is called alongside the maven publish process.

Reply:
Right now the release process is manual, and going to be manual because it involves signing with private keys, etc. which is hard to run in circle.ci etc without risking leaks.

Reply:
refering to the above thread, we want to run the release manually (at least for now). can you put all the steps of the python release in a script that can be run manually?

Reply:
- build release locally with `python setup.py bdist_wheel && python setup.py sdist`
- push to pypi `twine upload dist/*`